src/recipe/Baked.Recipe.Service/Communication/StatusCode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Baked.Communication;
+
+public enum StatusCode
+{
+    Success,
+    Redirection,
```

Comment:
redirection can be treated as success, it is specific to http, for communication it means success basically, let's remove it